packages/mobile/src/in-house-liquidity/index.ts

```diff
@@ -74,23 +76,10 @@ export const createPersonaAccount = async ({
   })
 }
 
-const getSerializedSignature = async ({
-  message,
-  accountMTWAddress,
-}: {
-  message: string
-  accountMTWAddress: string
-}): Promise<string> => {
-  const contractKit = await getContractKitAsync()
-  const accountWrapper = await contractKit.contracts.getAccounts()
-  const dataEncryptionKey = await accountWrapper.getDataEncryptionKey(accountMTWAddress)
-  return signWithDEK({ message, dataEncryptionKey })
```

Comment:
This actually didn't work-- the "dek" returned by the smart contract was the *public* key of the DEK pair, so signing with the public key was not useful. That's why the new DEK access pattern looks different.